analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -501,7 +501,11 @@ class Pip(
         }
 
         virtualEnvDir = createVirtualEnv(workingDir, projectPythonVersion)
-        installDependencies(workingDir, definitionFile, virtualEnvDir).requireSuccess()
+        val install = installDependencies(workingDir, definitionFile, virtualEnvDir)
```

Comment:
How about naming the `install` variable to something like `installDepsResult` or `installDepsProcessCapture` or just `installStatus`?

Reply:
Too long 😛 More seriously, I've sticked to the convention in this file to name the `ProcessCapture` variable after the command being run.